source/_integrations/bluetooth.markdown

```diff
@@ -25,6 +25,15 @@ While this integration is part of [`default_config:`](/integrations/default_conf
 bluetooth:
 ```
 
+## D-Bus and BlueZ are required on Linux
+
+For Bluetooth to function on Linux systems, the [D-Bus](https://en.wikipedia.org/wiki/D-Bus) socket must be accessible to Home Assistant. The Bluetooth adapter must be accessible to D-Bus and running [BlueZ](http://www.bluez.org/) >= 5.43.
+
+- Home Assistant Operating System: no additional steps are required. Version 8.4 or later is recommended for performance reasons.
```

Comment:
Just for clarity

```suggestion
- Home Assistant Operating System: no additional steps are required. Home Assistant OS version 8.4 or later is recommended for performance reasons.
```

Reply:
Should we trigger an repairs issue when we detect an older OS version being used with Bluetooth?

Reply:
I think thats a good idea.

Reply:
Added to my queue

Reply:
We should probably advise them to upgrade to a newer version of audio plugin once https://github.com/home-assistant/plugin-audio/pull/85 is done as well as all the bluetooth RSSI updates end up in the log and kill the cpu until that is fixed

Reply:
Supervisor plugins are kept in sync and up to date by the Supervisor. There is no need for us to detect that / inform the user about it.